prefpicker/templates/browser-fuzzing.yml

```diff
@@ -272,8 +272,15 @@ pref:
   dom.serviceWorkers.testing.enabled:
     default:
     - true
-  dom.streams.expose.ReadableStreamDefaultController:
+  dom.streams.writable_streams.enabled:
     default:
+    - false
```

Comment:
@pyoor let's use 'null' to indicate don't add to prefs file in the default case.